kmir/src/kmir/__main__.py

```diff
@@ -129,9 +131,15 @@ def exec_prove(
     # We provide configuration of which backend to use in a KMIR object.
     # `use_booster` is by default True, where Booster Backend is always used unless turned off
 
+    if claim_list:
+        claim_labels = get_claim_labels(kmir, spec_file)
+        print(*claim_labels, sep='\n')
+        sys.exit(0)
```

Comment:
Is it intended that this is left here? Or is this for testing?

Reply:
Intended for pretty printing of the list of the claim labels.